modules/registrars/openprovider/Controllers/System/DomainSuggestionsController.php

```diff
@@ -67,14 +74,56 @@ public function get($params)
         if (isset($suggestionSettings['sensitive']) && $suggestionSettings['sensitive'] == 'on')
             $args['sensitive'] = 1;
 
-        if (isset($suggestionSettings['suggestTlds']) && count($suggestionSettings['suggestTlds']) > 0)
+        if (isset($suggestionSettings['suggestTlds']) && count($suggestionSettings['suggestTlds']) > 0) {
             $args['tlds'] = array_map(function ($tld) {
                 return mb_substr($tld, 1);
             }, explode(',', $suggestionSettings['suggestTlds']));
+        }
+
+        $placementLogin = Configuration::get('placementPlusAccount');
```

Comment:
why we need to provide credentials here but not in PlacementPlus::getSuggestionDomain() if it's static?